go.mod

```diff
@@ -70,7 +70,7 @@ require (
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.1.1 // indirect
 	github.com/BurntSushi/toml v1.3.2 // indirect
 	github.com/PaesslerAG/gval v1.0.0 // indirect
-	github.com/ProtonMail/go-crypto v0.0.0-20230217124315-7d5c6f04bbb8 // indirect
+	github.com/ProtonMail/go-crypto v0.0.0-20230528122434-6f98819771a1 // indirect
```

Comment:
why all these updates?

Reply:
inherited from another commit and had to run go mod tidy

Reply:
can you roll these back? mod tidy doesn't produce anything for me

Reply:
yep, done! thanks